webpack/private/webpack_bundle.bzl

```diff
@@ -196,10 +197,20 @@ def _impl(ctx):
     executable = ctx.executable.webpack_exec_cfg
     execution_requirements = {}
 
+    no_copy_bin_inputs = []
     if ctx.attr.supports_workers:
         executable = ctx.executable.webpack_worker_exec_cfg
         execution_requirements["supports-workers"] = str(int(ctx.attr.supports_workers))
 
+        no_copy_bin_inputs.append(ctx.file._worker_js)
+
+        # TODO: get this path right!
+        if ctx.attr.use_execroot_entry_point:
+            env["JS_BINARY__ALLOW_EXECROOT_ENTRY_POINT_WITH_NO_COPY_DATA_TO_BIN"] = "1"
+            env["RULES_JS_WORKER"] = "/".join([".."] * len(ctx.label.package.split("/"))) + "/../../../external/burn/" + ctx.file._worker_js.short_path
```

Comment:
What's the `burn` dir?

Reply:
Temporary. Should have called the segment `TODO` :) I need to cleanup this path calculation before landing.

Reply:
✅